app/src/main/res/layout/ticket_detail_layout.xml

```diff
@@ -23,21 +23,34 @@
             android:layout_height="match_parent">
 
             <LinearLayout
+                style="@style/ItemPadding"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:orientation="vertical"
-                style="@style/ItemPadding">
+                android:orientation="vertical">
 
-                <TextView
-                    style="@style/TextAppearance.AppCompat.Headline"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_gravity="center_vertical"
-                    android:layout_marginEnd="@dimen/spacing_large"
-                    android:layout_marginRight="@dimen/spacing_large"
-                    android:padding="@dimen/spacing_extra_small"
-                    android:text='@{ ticket.name }'
-                    tools:text="Name" />
+                <RelativeLayout
```

Comment:
No use of RelativeLayout

Reply:
It is required so that the printer icon gets positioned to the right of the title.

Reply:
Use LinearLayout